File: leet-codes-review/fizzbuzz.js
let n = 15
let termo = 1
let list = []
while(list.length <= n){

    if ((termo % 5) == 0 && (termo % 3) == 0){
        list.push("fizzbuzz")
        termo = termo + 1
    }

    else if ((termo % 3) == 0){
        list.push("fizz")
        termo = termo + 1
    }

    else if ((termo % 5) == 0){
        list.push("buzz")
        termo = termo + 1
    }

    else {
        list.push(termo)
        termo = termo + 1
    }
}
console.log(list)

File: leet-codes-review/fizzbuzz.py
n = 15
termo = 1
list = []
while(len(list) <= n):

    if ((termo % 5) == 0 and (termo % 3) == 0):
        list.append("fizzbuzz")
        termo = termo + 1

    elif ((termo % 3) == 0):
        list.append("fizz")
        termo = termo + 1

    elif ((termo % 5) == 0):
        list.append("buzz")
        termo = termo + 1
        
    else:
        list.append(termo)
        termo = termo + 1

print(list)